doc/development/administration.rst

```diff
@@ -464,3 +470,95 @@ mature, some functionality will become public and others will become "private."
 Making a public API private should be treated as a deprecation.
 
 .. [#] During initial development, deprecations will target ``v0.Y``.
+
+
+.. _authorship:
+
+Authorship and attribution
+==========================
+
+The Celeritas team have specified this policy, to give proper credit in correct
+proportions to the many people who have contributed over the course of this
+project. All works by the Celeritas team should be added to the Celeritas doc
+repo and the `Celeritas Zotero database`_.
+
+.. _Celeritas Zotero database: https://www.zotero.org/groups/2380941/celeritas
+
+Code objects
+------------
+
+Each major and minor version of the code will be published in the `Celeritas
+Zenodo group`_ (see :ref:`release_process`). Creator/contributor listings are
+based on contributions since the *previous major release* (PRs since the "dev"
+tag indicates divergence from the previous major series), ordered by level of
+effort:
+
+- Creator: contributors whose pull requests are incorporated into the release,
+  ordered by number of PRs.
+- Contributor as "Editor": reviewers of PRs, ordered by number reviews
+- Contributor as "ProjectManager": code lead
+- Contributor as "ProjectMember": core team
+- Contributor as "ProjectLeader": core advisors
```

Comment:
Is the intent for names to be in general listed in 2 or 3 of those categories?  (Related question, did we already uploaded one set of those or it is still in the works)

Reply:
Originally I was going to deduplicate but after looking it makes more sense to have both roles: see https://zenodo.org/records/15256364